### PR TITLE
Adding nutrition major

### PR DESCRIPTION
**Pull request by nidhi-mylavarapu**
### Added Nutrition Sciences Major in CALS!

This PR is to add the Nutritional Sciences major to CP! 

### Test Plan 

- Double check major/course requirements with Cornell Registrar
- Add some courses and make sure updates appear appropriately in the left sidebar (test more complicated/important requirements if possible - specifically Advanced Electives, Nutrition Core)

### CODE REVIEW

**Review by zachary-kent — COMMENTED** [review]
It looks like there are some merge conflicts resulting from me adding importing new courses; you can resolve these by merging main into your branch, accepting all incoming changes, and then re-running `npm run req-gen`.

File: src/data/majors/ns.ts [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 2, 2, 1],
+    slotNames: ['CHEM 1570', 'CHEM 3570 or CHEM 3580', 'CHEM 3590 or CHEM 3600', 'CHEM 3530'],
+    minNumberOfSlots: 1,
+  },
+  {
+    name: 'Organic Chemistry Lab',
+    description: 'Either CHEM 2510 (Intro to Exp Org. Chem) or CHEM 3010 (Honors Exp. Chem I)',
+    source: 'https://courses.cornell.edu/preview_program.php?catoid=55&poid=28314',
+    checker: includesWithSubRequirements(['CHEM 2510', 'CHEM 3010']),
```
Comment (zachary-kent) [inline src/data/majors/ns.ts:54]:
```suggestion
    checker: includesWithSingleRequirement('CHEM 2510', 'CHEM 3010'),
```
We have a special function for where there is only one slot--can you make this change throughout this file?

File: src/data/majors/ns.ts [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+    perSlotMinCount: [3],
+    slotNames: ['Communcations Courses'],
+    minNumberOfSlots: 1,
+    fulfilledBy: 'courses',
+    additionalRequirements: {
+      'Courses must have at least 9 credits.': {
+        checker: [
+          (course: Course): boolean =>
+            nsSocialSciencesHumanities.some(
+              distribution => hasCategory(course, distribution) ?? false
```
Comment (zachary-kent) [inline src/data/majors/ns.ts:188]: Why do we need `?? false`?

**Review by zachary-kent — CHANGES_REQUESTED** [review]
Great work! Just a couple minor nits :)

**Review by andxu282 — APPROVED** [review]

**Review by andxu282 — APPROVED** [review]

**Pull request merged**